src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,181 @@
+import _ from 'lodash/fp'
+import { Fragment, useRef, useState } from 'react'
+import { div, h, span } from 'react-hyperscript-helpers'
```

Comment:
If you accept all span removals on this page, for your batch commit convenience:
```suggestion
import { div, h } from 'react-hyperscript-helpers'
```